mission/mission_core/missions/ScanTheCode_mission.py

```diff
@@ -0,0 +1,249 @@
+'''
+A simple mission template for creating new missions.
+
+The variables and functions defined here must exist in all mission classes.
+Comments are provided to explain the purpose of each variable and function and can be removed.
+'''
+
+from typing import Dict, Tuple
+from mission_core import PositionData
+from perception_core import CameraData, Results
+from itertools import groupby
+
+
+class SimpleMission:
+
+    # Define the initial perception commands here (if any)
+    # This dictionary must exist, but some commands are entered as examples
+    init_perc_cmd = {
+        "start": ["center"],
+        "stop": ["port", "starboard"],
+        "record": ["center"],
+        "stop_record": ["port", "starboard"],
+        "load_model": [("center", "ML_Resources/stcA.pt")],
+        "stop_model": ["port", "starboard"]
+    }
+
+    def __init__(self):
+        self.records = {"123":0, "132":0, "213":0, "231":0, "312":0, "321":0 }
+        self.debug = False
+        self.storageArray = []
+        self.stringAnalyzed = []
+        self.max_value = 0
+        self.light_pattern = []
+        
+        
+        
+
+    def __str__(self):
+        return self.__class__.__name__
+    
+    def mission_heartbeat(self):
+        '''
+        The heartbeat format can be found in the Team Handbook V3 Appendix D (Page 55).
+        The heartbeat must specify all relevant information about the mission BESIDES the following:
+         - Date
+         - Time
+         - Team ID
+         - Checksum
+        It must return a list of the required information in the correct order.
+        For example, Scan the Code would be: 
+        heartbeat = ["$RXCOD", self.light_pattern]
+        '''
+        heartbeat = []
```

Comment:
Make sure you fill out the list here. (All items in list must be a string)

Reply:
Resolved. Thank you